src/components/FeatureStore/featureStore.scss

```diff
@@ -4,6 +4,9 @@
   flex-grow: 1;
 
   .table-main {
+    display: flex;
+    flex-flow: column nowrap;
+    flex: 1;
```

Comment:
Why do we have styles for the main table exclusively in the feature store? If we need to add a similar panel to other pages, we will need to duplicate these styles. It's probably worth adding styles so that all tables could have a similar panel.